BackEnd/src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+spring:
```

Comment:
application.yml is preferred over application.properties

They are the exact same thing and serve the exact same purpose, but just have different syntax. As an example:

**application.properties**
`spring.application.name=TradingApp`
`spring.data.mongodb.uri=mongodb://localhost:27017/MarketStack`
`spring.profiles.active=default`

becomes this for application.yml

`spring:`
`  application:`
`    name: TradingApp`
`  data:`
`    mongodb:`
`      uri: mongodb://localhost:27017/MarketStack`
`  profiles:`
`    active: default`

See how it's the exact same thing but cleaner and leads to less redundant code?